backend/src/apiserver/server/artifact_server.go

```diff
@@ -85,10 +85,8 @@ func (s *ArtifactServer) ListArtifacts(ctx context.Context, r *apiv2beta1.ListAr
 	switch r.OrderBy {
```

Comment:
if `orderByAsc` is true anyways, why bother with a case-switch when we can just set `orderByAsc` to `false` if `desc`

Reply:
fixed